CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+## Contributing to Towhee
+
+Submitting issues, answering questions, and improving documentation are some of the many ways you can join our growing community. Regardless of how you contribute, please remember to be respectful towards your fellow Towheeians.
+
+### Contributing code
+
+The Towhee community maintains a list of [Good First Issues](https://github.com/towhee-io/towhee/contribute).
+
+### Sharing a new operator or pipeline
+
+Pipeline and operator contributions to our Towhee Hub are just as valued as artwork, code, and documentation contributions. If you have a new model, useful script, or an `x2vec` pipeline that you'd like to share with the Towhee community, [get in touch](towhee-team@zilliz.com) with us!
+
+### Submitting a new issue or feature request
+
+Please follow the templates we provide for submitting issues, bugs, and/or feature requests; try to avoid opening blank issues whenever possible.
```

Comment:
It can add link [submitting issues, bugs, and/or feature requests](https://github.com/towhee-io/towhee/issues/new/choose)

Reply:
please add another PR for these modifications